Apps/APN-UIKit/APN UIKit/APN UIKit/Model/Settings.swift

```diff
@@ -0,0 +1,27 @@
+import Foundation
+
+struct Settings {
+    var deviceToken: String
+    var trackUrl: String
+    var siteId: String
+    var apiKey: String
+    var bgQDelay: String
+    var bgQMinTasks: String
+    var isPushEnabled: Bool
+    var isTrackScreenEnabled: Bool
+    var isDeviceAttributeEnabled: Bool
+    var isDebugModeEnabled: Bool
```

Comment:
should we have default values for these?

Reply:
Default values are not needed here. This is just a struct as a data model to store values as done [here](https://github.com/customerio/customerio-ios/blob/08d4dc2ff883d3dffd5a21d3f7755ed776b44f44/Apps/APN-UIKit/APN%20UIKit/APN%20UIKit/View/SettingsViewController.swift#L39). All fields are mandatory to fill. I hope it answers your question.